tox.ini

```diff
@@ -79,7 +79,7 @@ commands =
 setenv = PYTHONHASHSEED=0
 changedir = doc
 commands =
-    sphinx-build -q -E -b doctest . {envtmpdir}/doctest {posargs}
```

Comment:
what does removing `-q` do?

Reply:
`-q` instructs `sphinx-build` to not send any output to stdout (it stands for 'quiet'). Errors shows up on stderr. 
I don't see the point of this - the doctest output isn't great but its still useful to look at it. It might have helped us catch the fact that most of our doctests weren't running at all.